packages/components/src/circular-option-picker/circular-option-picker.tsx

```diff
@@ -86,23 +86,22 @@ function ListboxCircularOptionPicker(
 		...additionalProps
 	} = props;
 
-	const compositeStore = useCompositeStore( {
-		focusLoop: loop,
-		rtl: isRTL(),
-	} );
-
-	const compositeContext = {
-		baseId,
-		compositeStore,
```

Comment:
The composite store can be accessed via the `Composite.Context` instead of this custom context